src/e2e-test/e2e-test.spec.js

```diff
@@ -0,0 +1,73 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+const supertest = require('supertest');
+const express = require('express');
+var fs = require("fs");
+var path  = require('path');
+var constants = require('../lib/constants/constants');
+const API_KEY_HEADER = "X-MS-CONVERSION-API-KEY";
+const apiKeys = ['123', '456'];
+var app = require("../routes")(express());
+
+// load testcases
+var cdaTestcases = require('./config/testcases-cda');
+var hl7v2Testcases = require('./config/testcases-hl7v2');
+var opTests = cdaTestcases.concat(hl7v2Testcases);
+
+describe('E2E test - FHIR data validation', function () {
+    before(function () {
+        app.setValidApiKeys(apiKeys);
+    });
+
+    opTests.forEach(t => {
+        it('should output a valid FHIR data with ' + t.dataFile + ' and ' + t.templateFile, function (done) {
+            var dataType = path.extname(t.dataFile);
+            var endpointURL = '';
+            var templateLocation = '';
+            var dataLocation = '';
+            var requestJson = {
+                templateBase64: "",
+                messageBase64: ""
+            };
+
+            if(dataType === '.hl7'){
+                endpointURL = '/api/convert/hl7v2';
+                templateLocation = constants.HL7V2_TEMPLATE_LOCATION;
+                dataLocation = constants.HL7V2_DATA_LOCATION;
+            }
+            else if(dataType === '.cda'){
+                endpointURL = '/api/convert/cda';
+                templateLocation = constants.CDA_TEMPLATE_LOCATION;
+                dataLocation = constants.CDA_DATA_LOCATION;
+            }
```

Comment:
Handle case for else?

Reply:
Thanks~I have modified it and it will throw an error when the data type is not supported.